src/get/getPerceivedDifference.ts

```diff
@@ -0,0 +1,123 @@
+import { rad2deg, deg2rad } from "../constants";
+import { LabaColor } from "../types";
+
+/**
+ * kl - grafic arts = 1; textiles = 2;
+ * kl - unity factor;
+ * kh - weighting factor;
+ */
+export interface DeltaE00Options {
+  kl: 1 | 2;
+  kc: number;
+  kh: number;
+}
```

Comment:
Where can I read about these options? I just wondering do we need them or not.

Reply:
You can find a little information about them. Most of the time they are equal to 1, they are weighting factors and as I understand means to provide some calibration for different ISO standards. Their origin is not from Delta E2000, but from Delta E94.

https://wisotop.de/assets/2017/DeltaE-%20Survey-2.pdf

Reply:
It just doesn't seem reasonable to me to provide options we can't even explain in the docs =) So let's simplify everything for now.